lib/generators/draft/resource/templates/controllers/controller.rb

```diff
@@ -1,13 +1,18 @@
 class <%= plural_table_name.camelize %>Controller < ApplicationController
   def index
-    @list_of_<%= plural_table_name.underscore %> = <%= class_name.singularize %>.all.order({ :created_at => :desc })
+    matching_<%= plural_table_name.underscore %> = <%= class_name.singularize %>.all
+
+    @list_of_<%= plural_table_name.underscore %> = matching_<%= plural_table_name.underscore %>.order({ :created_at => :desc })
```

Comment:
Isn't `plural_table_name` already underscored? I forget.

Reply:
Just did a check, `plural_table_name` does not underscore.

Reply:
I think it's a little inconsistent in `draft_generators`, but `plural_table_name` should already be underscored.

`plural_table_name` is just `table_name.pluralize` and `table_name` is underscored.

[`table_name`](https://api.rubyonrails.org/classes/Rails/Generators/NamedBase.html#method-i-table_name)

[`plural_table_name`](https://api.rubyonrails.org/classes/Rails/Generators/NamedBase.html#method-i-plural_table_name)

Reply:
That is interesting because when I was testing `plural_table_name` did not underscore, but it looks like in the docs it does say `table_name` is joining with `"_"`.  I wonder though because we do use `plural_table_name` throughout the generator with other methods such as  `.camelize`, `.humanize` and `title_case`, would the auto underscoring be an issue underscoring for `.camelize`?

Reply:
@jelaniwoods I moved our talk on `table_name` to a separate issue.   With that being moved out, is this ready to ship?